### ISSUE TITLE
publishToConfluence file not found

### ISSUE THREAD

**Issue by rdmueller** [docToolchain/docToolchain#143]
## Expected behavior and actual behavior.

the script tries to upload all referenced images as attachments. But if a referenced file is not there, the script throws an exception

Expected would be that it outputs a warning

**Comment by bodewig** [comment]
In my case a missing file should be an error, so I'm not sure whether outputting a warning would be enough.

**Comment by rdmueller** [comment]
you are right. we need proper error handling here. If it is part of a continuous build, it somehow has to handle this situation.

**Comment by bodewig** [comment]
The current behavior is "upload the page, try to upload all attachments, on error throw and exception and kill the build.", right?

What would "proper error handling" be? Verify the local file can be found and fail even before uploading the page? Have an option that suppresses failed uploads? I'm afraid I must be missing some important piece.

**Comment by rdmueller** [comment]
What about: try to upload all pages and images. On failure, skip to next item but memorize problem. When finished, throw problems.

This would not break the whole upload. Only parts... "One missing image, but the rest is updated"
